### PR TITLE
New project reloads watch timer

### PR DESCRIPTION
**Pull request by sohilpandya**
related to #53

### CODE REVIEW

**Comment by sohilpandya** [comment]
Now the timer on the watch is not affected when the user adds a new project, but the timer on the iPhone stops and is potentially attached to a completely different project. 
screen shot 2017-05-08 at 17 25 42

**Comment by sohilpandya** [comment]
As seen in the screenshot below, the timers are now in sync even when a new project is added but, if the row arrangement changes and the project move from pos. 1 to pos. 2 then the newly added task still has a timer attached to it. 

screen shot 2017-05-08 at 17 32 57

File: release-notes.md [hunk 1]
```diff
@@ -4,7 +4,17 @@ We'll be updating this document along with each update that is merged into maste
 
 We are now on Testflight :tada:, please forward us an email at hello@dwyl.io to request beta access
 
-## Version 1.1 (Build 5) _Current_
+## Version 1.1 (Build 7) _Current_
```
Comment (nelsonic) [inline release-notes.md:7]: love that the release notes are expanding. 
would quite like to have screenshots and even GIFs in here ... 🤔

**Review by nelsonic — APPROVED** [review]
@sohilpandya code looks great! Can't wait to try the new version! 👍

**Review by iteles — APPROVED** [review]
I still haven't been able to update my watchOS, but I'm pretty psyched to start testing this!

**Pull request merged**